exercises/yacht/canonical-data.json

```diff
@@ -0,0 +1,260 @@
+{  
```

Comment:
it would be preferable if all trailing spaces at the end of a line would be removed, since they don't change the meaning of the file and just take up space